### PR TITLE
[Feat]: A cool landing page! (UI + architectural major changes)

### PR DESCRIPTION
**Pull request by mkubdev**
### Describe the changes you've made

Hi! ✨ This is a lil update of the frontend of the repository. This PR has nothing related to web3 stuff yet.
I mostly did architectural changes:
* All app content is now in `src/` folder
* `src/utils` to build utility function re-usable accross the app
* `src/pages` 
* `src/components` 
* `src/styles`

Everything work as you already know from the `create-next-app` cli command.

## 💡 [Caveats](https://nextjs.org/docs/advanced-features/src-directory#caveats)
* `src/pages` will be ignored if pages is present in the root directory.
* Config files like `next.config.js` and `tsconfig.json`, as well as environment variables, **should be inside** the root directory, moving them to `src/` won't work. Same goes for the [public directory](https://nextjs.org/docs/basic-features/static-file-serving).

## 💄 Components and Layout

* Updated the `Footer` and `Navbar` component with a fancy design, and stored them in `src/components/navigation`.
* Added `GlobalLayout` component wich encapsulate our page content, so `Footer` and `Navbar` are still there on navigation without needing to declare it again.

> `Footer` component need rework, height is too high...

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation update.
- [x] This change requires a documentation update ⚠️⚠️⚠️ @WebXDAO/core-maintainer 

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 image | image |
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/components/Content.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+export default function Content() {
+  return (
+    <div className="h-full bg-white">
+      <div className="mx-auto max-w-6xl py-12 px-4 sm:px-6 lg:py-16 lg:px-8">
+        <div className="text-center">
+          <h1 className="bg-gradient-to-r from-purple-900 to-pink-700 bg-clip-text text-8xl font-extrabold text-transparent">
```
Comment (shuklaritvik06) [inline src/components/Content.js]:
```suggestion
          <h1 className="bg-gradient-to-r from-purple-900 to-pink-700 bg-clip-text  text-6xl sm:text-8xl  font-extrabold text-transparent">
```

**Review by shuklaritvik06 — COMMENTED** [review]
for mobile view you should decrease font size

File: src/components/navigation/Footer.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                <div className=" flex items-center justify-center space-x-4 text-sm font-semibold leading-6 text-slate-700 hover:text-slate-500">
+                  <a href="https://github.com/WebXDAO/nextjs-dapp-template">Github</a>
+                  <div className="h-4 w-px bg-slate-500/20"></div>
+                  <a href="https://github.com/WebXDAO/nextjs-dapp-template/commits/main">Changelog</a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="mt-6 border-t border-gray-200 pt-8">
+            <p className="text-sm font-semibold text-slate-700 xl:text-center">
```
Comment (shuklaritvik06) [inline src/components/navigation/Footer.js]:
```suggestion
            <p className="text-sm font-semibold text-slate-700 text-center">
```

File: src/components/navigation/Footer.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        </h2>
+        <div className="mx-auto max-w-7xl py-12 px-4 sm:px-6 lg:py-16 lg:px-8">
+          <div className="xl:grid xl:grid-cols-2 xl:gap-8">
+            <div className="space-y-8 xl:col-span-1">
+              <img className="h-10" src="/logo/logo.png" alt="Company name" />
+              <p className="text-base text-slate-700">
+                <span className="font-semibold">WebXDAO</span> is a community
+                that focus on the future of the web.
+              </p>
+              <div className="flex space-x-6">
```
Comment (shuklaritvik06) [inline src/components/navigation/Footer.js:44]:
```suggestion
              <div className="flex justify-center space-x-6">
```

File: src/components/navigation/Footer.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    <>
+      <footer className="bg-gray-100" aria-labelledby="footer-heading">
+        <h2 id="footer-heading" className="sr-only">
+          Footer
+        </h2>
+        <div className="mx-auto max-w-7xl py-12 px-4 sm:px-6 lg:py-16 lg:px-8">
+          <div className="xl:grid xl:grid-cols-2 xl:gap-8">
+            <div className="space-y-8 xl:col-span-1">
+              <img className="h-10" src="/logo/logo.png" alt="Company name" />
+              <p className="text-base text-slate-700">
```
Comment (shuklaritvik06) [inline src/components/navigation/Footer.js:40]:
```suggestion
              <p className="text-base  text-center text-slate-700">
```

**Review by shuklaritvik06 — CHANGES_REQUESTED** [review]
Here also

File: src/components/navigation/Navbar.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+                    src="/logo/logo.png"
+                    alt="WebXDAO"
+                  />
+                  <img
+                    className="hidden h-8 w-auto lg:block"
+                    src="/logo/logo.png"
+                    alt="WebXDAO"
+                  />
+                  <p className="text-slate-700 text-sm font-semibold">
+                    WebXDAO Community
```
Comment (vinzvinci) [inline src/components/navigation/Navbar.js]: Maybe we can remove Community here? Only **WebX DAO** add. CC @KukretiShubham

File: src/components/navigation/Footer.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+                    key={item.name}
+                    href={item.href}
+                    className="text-slate-700 hover:text-gray-500"
+                  >
+                    <span className="sr-only">{item.name}</span>
+                    <item.icon className="h-6 w-6" aria-hidden="true" />
+                  </a>
+                ))}
+                <div className=" flex items-center justify-center space-x-4 text-sm font-semibold leading-6 text-slate-700 hover:text-slate-500">
+                  <a href="https://github.com/WebXDAO/nextjs-dapp-template">Github</a>
```
Comment (vinzvinci) [inline src/components/navigation/Footer.js]: Always big letter H on GitHub

File: src/components/navigation/Navbar.js (hunk 5 above)
Comment (shuklaritvik06) [reply src/components/navigation/Navbar.js]: +1

File: src/components/navigation/Footer.js (hunk 6 above)
Comment (shuklaritvik06) [reply src/components/navigation/Footer.js]: +1

File: src/components/navigation/Navbar.js (hunk 5 above)
Comment (KukretiShubham) [reply src/components/navigation/Navbar.js]: +1

File: src/components/navigation/Footer.js (hunk 6 above)
Comment (KukretiShubham) [reply src/components/navigation/Footer.js]: +1

File: src/components/navigation/Navbar.js (hunk 5 above)
Comment (mkubdev) [inline src/components/navigation/Navbar.js]:
```suggestion
                    WebXDAO
```

Yup! Look better 🎉

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: src/components/navigation/Footer.js (hunk 6 above)
Comment (mkubdev) [inline src/components/navigation/Footer.js]:
```suggestion
                  <a href="https://github.com/WebXDAO/nextjs-dapp-template">GitHub</a>
```

Yes i always forgot the 'H'... 😂

btw, you should add a commit suggestion with this button :
image

**Review by shuklaritvik06 — APPROVED** [review]
LGTM! Thank you for your valuable contribution 🎉 ✨

**Pull request merged**